src/sentry/services/hybrid_cloud/rpc.py

```diff
@@ -97,11 +124,25 @@ def _extract_region_resolution(self) -> RegionResolution | None:
         return region_resolution
 
     def serialize_arguments(self, raw_arguments: ArgumentDict) -> ArgumentDict:
-        model_instance = self._model(**raw_arguments)
+        model_instance = self._parameter_model(**raw_arguments)
         return model_instance.dict()
 
     def deserialize_arguments(self, serial_arguments: ArgumentDict) -> pydantic.BaseModel:
-        return self._model.parse_obj(serial_arguments)
+        try:
+            return self._parameter_model.parse_obj(serial_arguments)
+        except Exception as e:
+            # TODO: Parse Pydantic's exception object(s) and produce more useful
```

Comment:
👍 
I have a feeling this meta data does exist in pydantic's validation, I can't recall where or how to retrieve it.

Reply:
I've seen it as well. At minimum, we can dump the exception message into the response body. As for fancier options, maybe.

Reply:
These seem like good errors for us to capture and alert on.